### ISSUE TITLE
import ndex2 outputs warnings in python3.9

### ISSUE THREAD

**Issue by coleslaw481** [ndexbio/ndex2-client#92]
Running `import ndex2` output these warnings in python 3.9
```
import ndex2
/Users/churas/src/ndex2-client/ndex2/nice_cx_network.py:1894: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  if mi.get("consistencyGroup" is not None):
/Users/churas/src/ndex2-client/ndex2/client.py:438: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if len(cx) is 0:
/Users/churas/src/ndex2-client/ndex2/client.py:527: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if len(cx) is 0:
```

**Comment by coleslaw481** [comment]
Fixed the 1st two in 3.5.1

**Comment by coleslaw481** [comment]
Finally fixed in 
   188eb38..8da1961  master -> master
